virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,50 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
+
+|====
+|Element
+|Description
 
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+|*Volumes project*
+|The current namespace.
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+|*Select volume to boot from*
+|Select a volume from the list to boot your VM from.
 
-// |*Add volume*
-// |Upload a new volume or use an existing PersistentVolumeClaim (PVC).
+|*Add volume*
+|Upload a new volume or use an existing PersistentVolumeClaim (PVC).
 
-// |*Filter* field
-// |Filter boot sources by operating system.
+|*Filter* field
+|Filter boot sources by operating system.
 
-// |Search field
-// |Search boot sources by keyword.
+|Search field
+|Search boot sources by keyword.
 
-// |*Select InstanceType*
-// |Select an instance type based on the number of CPUs and amount of memory available.
+|*Select InstanceType*
+|Select an instance type based on the number of CPUs and amount of memory available.
 
-// |*VirtualMachine details*
-// |Displays the current VM settings. You can change the name of the VM during this step.
+|*VirtualMachine details* pane
+|Displays the current VM settings. You can change the name of the VM during this step.
 
-// |*Advanced settings*
-// |Add an authorized SSH key to the VM.
+|*Advanced settings* pane
+|Add an authorized SSH key to the VM.
 
-// |*Start this VirtualMachine after creation* checkbox
-// |Checked by default. Uncheck this to prevent the new VM from starting after creation.
+|*Start this VirtualMachine after creation* checkbox
+|Checked by default. Uncheck this to prevent the new VM from starting after creation.
```

Comment:
Above, we removed "Uncheck" and replaced it with "Clear the *Start this VirtualMachine after creation* checkbox to prevent the VM from starting automatically.". 

Do we want to use the same language here?